### PR TITLE
Rolling Doc Changes July 2020

### PR DESCRIPTION
**Pull request by IronRico**
I added wording to detail the WGS 84 requirements to the Extent Import Features page.  A link to an outside resource was also made available in the description as well.

"{% hint style='info' %}
**Please Note:**  Imported data needs to utilize the widely used World Geodetic System 1984 (WGS 84) [WGS84] coordinate reference system.  This has longitude and latitude set in units of decimal degrees. For more details on this, please visit <a href="https://epsg.io/4326" target="_blank">the espg.io coordinate support site</a>.  Here you can find details for the defined unit measurements as well as popular file export options.
{% endhint %}"

  Carlos.P

### CODE REVIEW

File: reference/edit-window/metadata/extent-panels/features-import.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ __Drag & Drop__  The map supports dragging a file from your file explorer and dr
 
 ---
 
+{% hint style='info' %}
+**Please Note:**  Imported data needs to utilize the widely used World Geodetic System 1984 (WGS 84) [WGS84] coordinate reference system.  This has longitude and latitude set in units of decimal degrees. For more details on this, please visit <a href="https://epsg.io/4326" target="_blank">the espg.io coordinate support site</a>.  Here you can find details for the defined unit measurements as well as popular file export options.
```
Comment (jlblcc) [inline reference/edit-window/metadata/extent-panels/features-import.md:18]:
```suggestion
**Please Note:**  Imported data needs to utilize the widely used World Geodetic System 1984 (WGS 84) [WGS84] coordinate reference system.  This has longitude and latitude set in units of decimal degrees. For more details on this, please visit <a href="https://epsg.io/4326" target="_blank">the espg.io database website</a>.
```

Comment (jlblcc) [reply reference/edit-window/metadata/extent-panels/features-import.md:18]: See #24

Comment (jlblcc) [reply reference/edit-window/metadata/extent-panels/features-import.md:18]: @IronRico I'll let you handle these inline suggestions - don't want to commit and mess up your workflow.

File: reference/edit-window/metadata/distribution-panels/onlineOption.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+### Online Transfer Option 
+
+![Online Transfer Option Edit Window](/assets/reference/edit-objects/metadata/distribution/OnlineTransferOption.PNG)
```
Comment (jlblcc) [inline reference/edit-window/metadata/distribution-panels/onlineOption.md:3]: This should be moved to the [include file](https://github.com/adiwg/mdEditor-doc/blob/master/reference/include-objects/onlineResource-obj.md) and that file should be included here.

File: tutorial/editor-window-parts/secondary-sidebar/fast-scrolling.md [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
 
-To assist with quickly displaying the portion of the page you need, the <span class="md-window">Fast Scrolling</span> feature was added to the <span class="md-window">Secondary Sidebar</span>.  This facility provides links to quickly scroll the panel you need to the top of the <span class="md-window">Edit Window</span>.  In addition, the window's current scroll position is highlighted in the <span class="md-window">Secondary Sidebar</span>.  
+To assist with quickly displaying the portion of the page you need, the <span class="md-window">Fast Scrolling</span> feature was added to the <span class="md-window">Secondary Sidebar</span>.  This facility provides links to quickly scroll the panel you need to the top of the <span class="md-window">Edit Window</span>.  
+
+
+![Secondary Sidebar Fast Scrolling Facility](/assets/tutorial/fast-scrolling-noHighlightSideBar.png)
```
Comment (jlblcc) [inline tutorial/editor-window-parts/secondary-sidebar/fast-scrolling.md:10]: Missing the caption:

```suggestion
![Secondary Sidebar Fast Scrolling Facility](/assets/tutorial/fast-scrolling-noHighlightSideBar.png){caption}
```

Comment (jlblcc) [reply tutorial/editor-window-parts/secondary-sidebar/fast-scrolling.md:10]: See #27

**Review by jlblcc — DISMISSED** [review]
See inline code suggestions.

File: reference/edit-window/metadata/extent-panels/features-import.md (hunk 1 above)
Comment (jlblcc) [reply reference/edit-window/metadata/extent-panels/features-import.md:18]: Didn't test to see if that `<sup>` will render correctly in the markdown. If not just remove it. We should add some css for `<a class="external">` to visually indicate an external link opening a new tab. Example:

image

File: reference/edit-window/metadata/distribution-panels/onlineOption.md (hunk 2 above)
Comment (IronRico) [reply reference/edit-window/metadata/distribution-panels/onlineOption.md]: I updated this file with the following:

" {% include "../../../include-objects/onlineTransferOption.md" %} "

I tested this locally, however; I would like a second opinion on its use.

**Comment by IronRico** [comment]
For this issue: "Remove "Attribute" from "Primary/Foreign" elements #33"

I used a new screenshot.  Please review for any considered issues:

---

AtrributeNameAnnotations

File: reference/edit-window/metadata/distribution-panels/onlineOption.md (hunk 2 above)
Comment (IronRico) [reply reference/edit-window/metadata/distribution-panels/onlineOption.md]: Before I resolve this conversation, I want to verify if you spot any issue with the change.

**Comment by IronRico** [comment]
I worked to correct the navigation issues found in the Distribution section of the Summary.md file.

I replaced these with the following:

- [Online Transfer Option](reference/edit-window/metadata/distribution-panels/ onlineTransferOption.md)
- [Transfer Frequency] (reference/edit-window/metadata/distribution-panels/ timeDuration-obj.md)

I tested these items locally and found them to be working as intended.

**Comment by IronRico** [comment]
I reviewed and commented on the changes requested.  None were declined with all being implemented.

**Comment by IronRico** [comment]
**Updates:**
General Settings Info #34
---
I made some of the proposed changes on my own repo.  Please see the screenshot below.

87886625-4f658080-c9e4-11ea-9bc2-638ebe3988c0


---

Please advise on what you think works or does not work.  The wording is a bit of a place holder for the time being so feel free to advise on any concepts that need to be addressed.

**Review by IronRico — COMMENTED** [review]
Reviewed the changes and found nothing to be out of range or in any form of extreme change.

**Review by IronRico — COMMENTED** [review]
Minor changes were approved and implemented as requested.

File: tutorial/editor-window-parts/secondary-sidebar/fast-scrolling.md [hunk 4]
```diff
@@ -1,23 +1,21 @@
 # Tutorial -- Fast Scrolling
---- 
+
+---
 
 Some <span class="md-window">Edit Window</span> sections are too long to fit on a standard monitor and require some scrolling to see all the elements of the page.  The Edit Window can get even longer as you continue to add your metadata.  This can make it inconvenient to locate just the information you're looking for by visually searching the record.
 
-To assist with quickly displaying the portion of the page you need, the <span class="md-window">Fast Scrolling</span> feature was added to the <span class="md-window">Secondary Sidebar</span>.  This facility provides links to quickly scroll the panel you need to the top of the <span class="md-window">Edit Window</span>.  In addition, the window's current scroll position is highlighted in the <span class="md-window">Secondary Sidebar</span>.  
+To assist with quickly displaying the portion of the page you need, the <span class="md-window">Fast Scrolling</span> feature was added to the <span class="md-window">Secondary Sidebar</span>.  This facility provides links to quickly scroll the panel you need to the top of the <span class="md-window">Edit Window</span>.
```
Comment (hamaier) [inline tutorial/editor-window-parts/secondary-sidebar/fast-scrolling.md:7]:
```suggestion
To assist with quickly displaying the portion of the page you need, the <span class="md-window">Fast Scrolling</span> feature on the <span class="md-window">Secondary Sidebar</span> provides links to quickly scroll the panel you need to the top of the <span class="md-window">Edit Window</span>.  
```

File: reference/edit-window/metadata/distribution-panels/onlineOption.md (hunk 2 above)
Comment (hamaier) [reply reference/edit-window/metadata/distribution-panels/onlineOption.md]: It looks like the "Online Option" link on the "[Transfer Option](https://ironrico.github.io/mdEditor-doc/reference/edit-window/metadata/distribution-panels/transferOption.html)" page is pointing to https://ironrico.github.io/mdEditor-doc/reference/edit-window/metadata/distribution-panels/onlineTransferOption.md (the current page).

The table of contents "Online Options" points to: https://ironrico.github.io/mdEditor-doc/reference/include-objects/onlineTransferOption.html, which does display the correct information.

File: reference/import-window/clear-cache.md [hunk 5]
```diff
@@ -1,16 +1,21 @@
 # Reference -- Import Records 
+
 ### Clearing Browser Cache
---- 
 
-When importing metadata records to mdEditor there are times when completely clearing the browser cache can be helpful.  If your import is replacing items of each type (<span class="md-panel">Metadata Record</span>, <span class="md-panel">Contact</span>, and <span class="md-panel">Dictionary</span>) mdEditor will effectively be clearing browser cache for you.  In other instances you may need to do this yourself.  
+---
+
+When importing metadata records to mdEditor there are times when completely clearing the browser cache can be helpful. If your import is replacing items of each type (<span class="md-panel">Metadata Record</span>, <span class="md-panel">Contact</span>, and <span class="md-panel">Dictionary</span>) mdEditor will effectively be clearing browser cache for you. In other instances, you may need to do this yourself.
```
Comment (hamaier) [inline reference/import-window/clear-cache.md:7]:
```suggestion
When importing metadata records to mdEditor there are times when completely clearing the editor storage cache can be helpful. If your import is replacing items of each type (<span class="md-panel">Metadata Record</span>, <span class="md-panel">Contact</span>, and <span class="md-panel">Dictionary</span>) mdEditor will effectively be clearing browser cache for you. In other instances, you may need to do this yourself. 
```

**Review by hamaier — CHANGES_REQUESTED** [review]
We should be referring to the "mdEditor storage cache" instead of the "browser cache" to avoid users confusing this with what they typically think of when clearing the cache (i.e. losing their browsing history or conversely, losing their mdEditor records if they clear their history)

**Review by IronRico — COMMENTED** [review]
All changes were reviewed.

**Review by hamaier — APPROVED** [review]

**Review by hamaier — APPROVED** [review]

**Review by hamaier — APPROVED** [review]

**Review by hamaier — APPROVED** [review]

**Review by hamaier — APPROVED** [review]

**Comment by hamaier** [comment]
Documentation updates

**Pull request merged**